### PR TITLE
Revise RHDH operator missing required app config fields disclaimer

### PR DESCRIPTION
**Pull request by michael-valdron**
### What does this PR do?:

Revises RHDH operator missing required app config fields disclaimer to make clear that the following field **must** be set under a tied app config if not already:

- `.app.baseUrl`
- `.backend.baseUrl`
- `.backend.cors.origin`

Also provides example for how this should be done.

### Which issue(s) this PR fixes:

https://issues.redhat.com/browse/DEVAI-235

### PR acceptance criteria:
Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened and linked to this PR, if they are not in the PR scope due to various constraints.

- [ ] Tested and Verified

  

- [X] Documentation (READMEs, Product Docs, Blogs, Education Modules, etc.)

### CODE REVIEW

**Review by thepetk — COMMENTED** [review]
lgtm in general. There's a related information [item](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-config-rhdh-custom-resource_admin-rhdh) in RHDH docs too btw. Not sure if we want to mention for further reading.

**Comment by michael-valdron** [comment]
> lgtm in general. There's a related information [item](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-config-rhdh-custom-resource_admin-rhdh) in RHDH docs too btw. Not sure if we want to mention for further reading.

I could refer to this for more details at the end. @elsony @yangcao77 wdyt?

**Review by yangcao77 — APPROVED** [review]

**Comment by yangcao77** [comment]
+1 on @thepetk 's comment. we can reference RHDH doc as well.

**Comment by michael-valdron** [comment]
>> lgtm in general. There's a related information [item](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-config-rhdh-custom-resource_admin-rhdh) in RHDH docs too btw. Not sure if we want to mention for further reading.
>
> +1 on @thepetk 's comment. we can reference RHDH doc as well.

https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/25/commits/a325d1b36aa0e0be3da87414e337950f140995f4 addresses this.

File: docs/RHDH-CONFIG.md [hunk 1]
```diff
@@ -308,11 +308,27 @@ You are able to store these values in environment variables. Set the following t
...
+    app:
+      title: "Red Hat Developer Hub for AI Software Templates"
+      baseUrl: https://backstage-ai-rh-developer-hub-ai-rhdh.apps.example-cluster.devcluster.openshift.com
+    backend:
+      baseUrl: https://backstage-ai-rh-developer-hub-ai-rhdh.apps.example-cluster.devcluster.openshift.com
+      cors:
+        origin: https://backstage-ai-rh-developer-hub-ai-rhdh.apps.example-cluster.devcluster.openshift.com
+```
+
+**Note**: Having these fields set under any app config ConfigMap tied to the existing RHDH instance should work. See [Configuring the Developer Hub Custom Resource](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.1/html/administration_guide_for_red_hat_developer_hub/assembly-install-rhdh-ocp#proc-config-rhdh-custom-resource_admin-rhdh) for further details about setting up the RHDH app config.
```
Comment (maysunfaisal) [inline docs/RHDH-CONFIG.md]: its for RHDH 1.1 but not there for 1.2

Comment (thepetk) [reply docs/RHDH-CONFIG.md]: Yeah, is slightly changed. Should we go then with https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/administration_guide_for_red_hat_developer_hub/assembly-add-custom-app-file-openshift_admin-rhdh#proc-add-custom-app-config-file-ocp-operator_admin-rhdh? Seems to me that it gives info for the above issue. WDYT?

Comment (michael-valdron) [reply docs/RHDH-CONFIG.md]: @maysunfaisal Should be resolved by https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/25/commits/4853537afbe8e23c491b64fed8d1cf5a57b803fb, wdyt?

**Review by maysunfaisal — APPROVED** [review]

**Review by thepetk — APPROVED** [review]

**Pull request merged**